cherrypy/test/test_static.py

```diff
@@ -215,13 +215,13 @@ def test_config_errors(self):
         self.assertErrorPage(500)
         if sys.version_info >= (3, 3):
             errmsg = (
-                'TypeError: staticdir\(\) missing 2 '
+                r'TypeError: staticdir\(\) missing 2 '
```

Comment:
Making docstrings in _cpreqbody.py raw strings don't change they. If backslashes were needed in non-raw strings, it should be kept in raw strings.

The docstring in helper.py is different case. Backslashes following by newlines was not literal backslashes, they escaped newlines. After making the docstring a raw string they become literal backslashes. I'm not sure this change was intention, and I don't know what content (before or after the change) is more correct.